### PR TITLE
WIP: Mega Pokemon stats

### PR DESCRIPTION
**Pull request by anapple96**
Cut Mega from #586 
This PR will add the stat for all current Mega Pokemons.
They're unobtainable for now
(more will be added later)

### CODE REVIEW

File: src/scripts/pokemons/PokemonList.ts [hunk 1]
```diff
@@ -13370,7 +13370,7 @@ const pokemonList: {
             'type': [PokemonType.Grass],
             'eggCycles': 20,
             'levelType': LevelType.mediumslow,
-            'exp': 230,
+            'exp': 236,
```
Comment (RedSparr0w) [inline src/scripts/pokemons/PokemonList.ts]:
```suggestion
            'exp': 230,
```
Could you change this back to 230:
https://bulbapedia.bulbagarden.net/wiki/Tsareena_(Pok%C3%A9mon)
image

**Comment by anapple96** [comment]
oops, clicking the wrong mon

File: src/scripts/pokemons/PokemonList.ts (hunk 1 above)
Comment (anapple96) [reply src/scripts/pokemons/PokemonList.ts]: fixed!

File: src/scripts/pokemons/PokemonList.ts [hunk 2]
```diff
@@ -14427,6 +14427,57 @@ const pokemonList: {
...
+            'catchRate': 3,
+            'base': {
+                'hitpoints': 105,
+                'attack': 180,
+                'specialAttack': 180,
+                'defense': 100,
+                'specialDefense': 100,
+                'speed': 115,
+            },
+        },
```
Comment (anapple96) [inline src/scripts/pokemons/PokemonList.ts]: Daddy

**Pull request closed without merging**